Lib/wave.py

```diff
@@ -150,7 +150,11 @@ def initfp(self, file):
                 self._nframes = chunk.chunksize // self._framesize
                 self._data_seek_needed = 0
                 break
-            chunk.skip()
+            try:
+                chunk.skip()
+            except RuntimeError:
```

Comment:
Why this was added? Tests are passed without this change.

Reply:
I don't think they actually are. It's been a while so looking at this again I get the following issue when running these tests without this fix. 

```
david@lithium ~/P/P/c/L/test>pso-34088>../../python.exe test_sndhdr.py          18:22:08
E..
======================================================================
ERROR: test_badinputs (__main__.TestFormats)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_sndhdr.py", line 35, in test_badinputs
    what = sndhdr.what(filename)
  File "/Users/david/Projects/PYTHON/cpython/Lib/sndhdr.py", line 54, in what
    res = whathdr(filename)
  File "/Users/david/Projects/PYTHON/cpython/Lib/sndhdr.py", line 63, in whathdr
    res = tf(h, f)
  File "/Users/david/Projects/PYTHON/cpython/Lib/sndhdr.py", line 163, in test_wav
    w = wave.open(f, 'r')
  File "/Users/david/Projects/PYTHON/cpython/Lib/wave.py", line 510, in open
    return Wave_read(f)
  File "/Users/david/Projects/PYTHON/cpython/Lib/wave.py", line 164, in __init__
    self.initfp(f)
  File "/Users/david/Projects/PYTHON/cpython/Lib/wave.py", line 153, in initfp
    chunk.skip()
  File "/Users/david/Projects/PYTHON/cpython/Lib/chunk.py", line 160, in skip
    self.file.seek(n, 1)
  File "/Users/david/Projects/PYTHON/cpython/Lib/chunk.py", line 113, in seek
    raise RuntimeError
RuntimeError

----------------------------------------------------------------------
Ran 3 tests in 0.010s

FAILED (errors=1)
```

Reply:
I believe at the time I thought that this file should have reported the runtime error if it failed to skip through a wav file that was poorly formatted. 

I'm not very experienced at being a principled programmer, @serhiy-storchaka Where do you think is the best place to catch this error or deal with the tests failing on the bad input files?